code/planning/src/planning/preplanning_trajectory.py

```diff
@@ -0,0 +1,522 @@
+from xml.etree import ElementTree as eTree
+import help_functions
+
```

Comment:
i have troubles reviewing and testing this file because i dont know where to start here and more importantly how to start this when putting the node into the launch file didnt work for me. please add some form of information on how to use this / reference where its from. I assume WIP on this file?